### ISSUE TITLE
Please migrate from Dart to React.

### ISSUE THREAD

**Issue by philippeBchn** [pampul/ShadowHunters#3]
8 YEARS OF WAITING FOR THIS AMAZING PROJECT AND...

![giphy](https://user-images.githubusercontent.com/6814372/144826657-bbbd7990-05c2-42f0-a4dd-3829a9ea8b4c.gif)

**Comment by AcelisWeaven** [comment]
Yeah seriously, what's going on? Has the maintainer been trapped in another dimension or something?
